tensorflow/lite/micro/kernels/README.md

```diff
@@ -0,0 +1,5 @@
+# Info
+
+*   [Porting Ops from Lite to Micro](./PORTING.md) explains, step-by-step, the
```

Comment:
let's rename and move to docs/porting_reference_ops.md

porting.md is too generic since we also have docs on porting TFLM to a new target etc.